scripts/send_erc20_packet.js

```diff
@@ -0,0 +1,48 @@
+// This is for emitting SendPacket event directly from Axon to initial the IBC communication
+
+async function main() {
+  // First provider account.
+  const [sender] = await web3.eth.getAccounts();
```

Comment:
Would it be better if the sender and amount can be specified at runtime?

Reply:
I would love too, so do you suggest putting them in `.env` file before running this script?